packages/release/release.spec

```diff
@@ -114,7 +118,13 @@ Requires: %{_cross_os}util-linux
 install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
 install -p -m 0644 %{S:11} %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
 
-install -d %{buildroot}%{_cross_libdir}/repart.d
+install -d %{buildroot}%{_cross_libdir}/repart-data-a.d/
+install -p -m 0644 %{S:94} %{buildroot}%{_cross_libdir}/repart-data-a.d/80-data-a.conf
+
+install -d %{buildroot}%{_cross_libdir}/repart-data-b.d/
+install -p -m 0644 %{S:95} %{buildroot}%{_cross_libdir}/repart-data-b.d/80-data-b.conf
```

Comment:
These repart configs are all identical now, right? So we can just use the one in `/usr/lib/repart.d/80-local.conf`.

Reply:
Ah you're right!